frontend/vite.config.ts

```diff
@@ -82,7 +82,7 @@ const insertDevCSPPlugin: Plugin = {
         "<head>",
         `<head>
         <!-- DEV-ONLY CSP - when making changes here, also update the CSP header in http_service.go (without the nonce!) -->
-        <meta http-equiv="Content-Security-Policy" content="default-src 'self' ${DEVELOPMENT_NONCE}; img-src 'self' https://uploads.getalby-assets.com https://getalby.com;"/>`
+        <meta http-equiv="Content-Security-Policy" content="default-src 'self' ${DEVELOPMENT_NONCE}; img-src 'self' https://uploads.getalby-assets.com https://getalby.com; connect-src 'self' https://api.getalby.com"/>`
```

Comment:
This enables fetching the invoice data from ln address via our [ln-proxy](https://github.com/getAlby/lightning-address-details-proxy)

Reply:
@im-adithya can you check the comment above this, we also need to change the CSP header in http_service.go

Reply:
Whoops sorry, completely missed it, adding now 👍